src/lib/services/client-metrics/metrics-service-v2.ts

```diff
@@ -183,17 +187,26 @@ export default class ClientMetricsServiceV2 {
         featureName: string,
         hoursBack: number = 24,
     ): Promise<IClientMetricsEnv[]> {
-        const metrics = this.flagResolver.isEnabled('extendedUsageMetrics')
-            ? await this.clientMetricsStoreV2.getMetricsForFeatureToggleV2(
-                  featureName,
-                  hoursBack,
-              )
-            : await this.clientMetricsStoreV2.getMetricsForFeatureToggle(
-                  featureName,
-                  hoursBack,
-              );
-
-        const hours = generateHourBuckets(hoursBack);
+        let hours: HourBucket[];
+        let metrics: IClientMetricsEnv[];
+        if (this.flagResolver.isEnabled('extendedUsageMetrics')) {
```

Comment:
new and old path clearly separated to have an easy mechanism to turn new metrics on and off